client/tests/test_client.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+from client import models
+from client.models import Client
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
+
+
+USER_NAME_TEST = 'liorsil'
+FIRST_NAME_TEST = 'lior'
+LAST_NAME_TEST = 'silber'
+PASSWORD_TEST = '1234qwer'
+AREA_TEST = 'yuvalim'
+
+
+@pytest.fixture()
+def client0():
+    return models.Client(user_name=USER_NAME_TEST,
+                         first_name=FIRST_NAME_TEST,
+                         last_name=LAST_NAME_TEST,
+                         password=PASSWORD_TEST,
+                         area=AREA_TEST)
+
+
+@pytest.fixture
+def saved_client0(client0):
+    client0.save()
+    return client0
+
+
+class TestClientModel:
+    @pytest.mark.django_db()
+    def test_save_client(self, client0):
+        assert client0 not in models.Client.objects.all()
+        client0.save_client()
+        assert client0 in models.Client.objects.all()
+
+    @pytest.mark.django_db()
+    def test_delete_client(self, saved_client0):
+        assert saved_client0 in Client.objects.all()
+        saved_client0.delete_client()
+        assert saved_client0 not in Client.objects.all()
+
+    @pytest.mark.django_db()
+    def test_delete_not_exist_clinet(self, client0):
+        assert client0 not in Client.objects.all()
+        with pytest.raises(ObjectDoesNotExist):
+            client0.delete_client()
+
+    @pytest.mark.django_db()
+    def test_get_all_clients(self):
```

Comment:
This test returns true all the time except if you dont have clients,what is the point?